build-support/scripts/proto-gen.sh

```diff
@@ -116,7 +116,7 @@ function main {
       return 1
    fi
 
-   BUILD_TAGS=$(sed -e '/^[:space:]*$/,$d' < "${proto_path}" | grep '// +build')
+   BUILD_TAGS=$(sed -e '/^[[:space:]]*$/,$d' < "${proto_path}" | grep '// +build')
```

Comment:
The `[:space:]` pattern does not work on linux. Could someone on macOS check this? If it doesn't work with this change we can use something like `[ \t]` I think.

Reply:
Seems to still work for me on macOS.